app/wasm/test/custom_msg_test.go

```diff
@@ -2,6 +2,7 @@ package wasm
 
```

Comment:
Can we make this a `wasm_test` package as well as all of the other test files:
- `custom_msg_test.go`
- `custom_query_test.go`
- `messages_test.go`
- `queries_test.go`

Let's not block merge on this but creating an issue for that would be great!

Reply:
What is the reasoning here?

Basically, the default is usually the same package, and there are valid reasons to make it a separate `wasm_test` package, but it is good to discuss which one to prefer